skimage/morphology/rolling_ball.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+from itertools import product
+
+from skimage.util import invert, view_as_windows
+
+
+def rolling_ball(image, radius=50, white_background=False,
```

Comment:
```suggestion
def rolling_ball(image, *, radius=50, white_background=False,
```

Is giving a default to `radius` actually meaningful or is this value expected to be highly specific to each image? In that case I'd move the `asterisk` one parameter to the right and remove the default.

Reply:
I guess it is specific to the image; I used `50`, because it is the default value in ImageJ, which I used as a reference when I started implementing this.

I also like the ability to do `filtered_image = rolling_ball(image)` without worrying too much about the specifics initially.

Reply:
I think 50 is fine, and I agree, let's make it keyword-only. (btw @lagru, required parameters can also be keyword only... ;)